docs/fields.md

```diff
@@ -0,0 +1,73 @@
+# aeacus
+
+## Fields
+
+This is a list of configuration fields that are required when creating `aeacus`. Well, most of them are required.
+
+**name**: Image name, primarily used to organize remote scoring.
+
+> **Note!** This field is not mandatory if you use local scoring.
+
+```
+name = "ubuntu-18-dabbingdabbers"
+```
+
+**title**: Round title, as seen in the scoring report and README.
+
+```
+title = "CyberPatio Practice Round 18"
+```
+
+**os**: Name of the operating system, as seen in the README.
+
+```
+os = "TempleOS 5.03"
+```
+
+**user**: Main user of the image.
+
+```
+user = "sysadmin"
+```
+
+**remote**: Address of remote server for scoring. If remote scoring is enabled, aeacus will refuse to score the image unless a connection to the server can be established.
+
+```
+remote = "8.8.8.8"
+```
+
+**password**: Password used for encrypting remote reporting traffic.
+
+```
+password = "H4!b5at+kWls-8yh4Guq"
+```
+
+**local**: Enables (technically, disables) remote scoring. If no remote address is specified, this will automatically be set to true.
```

Comment:
Can you just change this to "Disables remote scoring"

Reply:
Not according to the example scoring.

Reply:
Yeah, I was concerned about verbosity. I’ll rewrite in a bit.